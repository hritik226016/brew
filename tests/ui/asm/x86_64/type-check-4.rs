//@ only-x86_64
//@ compile-flags: -C target-feature=+avx512f
//@ check-pass

use std::arch::x86_64::{_mm256_setzero_ps, _mm_setzero_ps};
use std::arch::{asm, global_asm};

fn main() {}

static S: i32 = 1;
const fn const_foo(x: i32) -> i32 {
    x
}
const fn const_bar<T>(x: T) -> T {
    x
}
global_asm!("{}", const S);
global_asm!("{}", const const_foo(0));
global_asm!("{}", const const_foo(S));
global_asm!("{}", const const_bar(0));
global_asm!("{}", const const_bar(S));
