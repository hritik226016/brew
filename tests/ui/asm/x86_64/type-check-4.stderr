warning: unstable feature specified for `-Ctarget-feature`: `avx512f`
   |
   = note: this feature is not stably supported; its behavior can change in the future

warning: 1 warning emitted

